integration_tests/print_02.py

```diff
@@ -0,0 +1,19 @@
+from ltypes import i32, f64
+
+# Test: Printing ListConstant
+def f():
+    a: list[str] = ["ab", "abc", "abcd"]
```

Comment:
What if the list is nested? Something like, `anested: list[list[i32]]`

Can you add a test for the same?

Reply:
Thank you for sharing. I think this pass does not support it currently.

Reply:
I added the support for nested lists and also tests for the same.